core/src/main/java/org/ehcache/core/Ehcache.java

```diff
@@ -122,6 +135,9 @@ public Ehcache(CacheConfiguration<K, V> configuration, final Store<K, V> store,
     }
   }
 
+  /**
+   * {@inheritDoc}
+   */
   @Override
```

Comment:
I tend to use them as a way of confirming I do not want to specify any additional information.